sync_issues_to_jira/sync_issue.py

```diff
@@ -71,8 +71,11 @@ def handle_issue_closed(jira, event):
     # issues often get closed for the wrong reasons - ie the user
     # found a workaround but the root cause still exists.
     issue = _leave_jira_issue_comment(jira, event, "closed", False)
-    # Sets value of custom GitHub Issue field to Closed
-    issue.update(fields={'customfield_12100': {'value': 'Closed'}}) 
+    try :
+        # Sets value of custom GitHub Issue field to Closed
+        issue.update(fields={'customfield_12100': {'value': 'Closed'}})
+    except JIRAError as error:
```

Comment:
@tom-borcin Are we sure this is a `JIRAError` exception?

It seems to fail with an `AttributeError` error,  https://github.com/espressif/esp-matter/actions/runs/5044110229/jobs/9046736828#step:4:222

Reply:
I tested it on my own fork and it worked: tom-borcin/github-actions/sync_issues_to_jira@fix/test_custom_field

Linked action fails with: `AttributeError: 'NoneType' object has no attribute 'update'
[222](https://github.com/espressif/esp-matter/actions/runs/5044110229/jobs/9046736828#step:4:223)
}` on this GitHub issue: https://github.com/espressif/esp-matter/issues/396

GH issue wasn't mirrored to Jira and script just line before on [L73](https://github.com/espressif/github-actions/blob/master/sync_issues_to_jira/sync_issue.py#L73) returns None (should return mirrored Jira issue). Calling update on None type causes AttributeError.

Reply:
@tom-borcin I'm sorry, but I'm a bit confused by this explanation ...

> ..... Calling update on None type causes `AttributeError`.

So in that case we should change it to :
```python
try :
	# Sets value of custom GitHub Issue field to Closed
	issue.update(fields={'customfield_12100': {'value': 'Closed'}})

except AttributeError as error:
	...
```

Or am I missing something? Thank you

@kumekay PTAL as well ... many thanks!